os-support/os_3.0.0.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
Nitpick. I would use a more generic name (for instance, `pc_support`) for the directory.